### PR TITLE
Clean up tests

### PR DESCRIPTION
**Pull request by emlun**
The integration tests now reliably pass in both Linux and Mac!

### CODE REVIEW

File: ykman/device.py [hunk 1]
```diff
@@ -180,6 +184,10 @@ def use_transport(self, transport):
         assert dev.mode == my_mode
         return dev
 
+    def close(self):
+        logger.debug('close')
```
Comment (dagheyman) [inline ykman/device.py]: Remove this log message or improve it a bit.

File: ykman/driver.py [hunk 2]
```diff
@@ -64,3 +68,6 @@ def read_capabilities(self):
     def guess_version(self):
         # Second arg is True if the version is certain, False if not.
         return (0, 0, 0), False
+
+    def close(self):
+        logger.debug('close')
```
Comment (dagheyman) [inline ykman/driver.py]: See above (about log msg)

File: test/test_external_libs.py [hunk 3]
```diff
@@ -1,10 +1,7 @@
 import unittest
-import os
 from test.util import ykman_cli
 
 
-@unittest.skipIf(
-    os.environ.get('INTEGRATION_TESTS') != 'TRUE', 'INTEGRATION_TESTS != TRUE')
```
Comment (dagheyman) [inline test/test_external_libs.py]: Are you sure about this? Shouldn't the python only unittests be able to run without the libraries in place?

**Comment by dagheyman** [comment]
Very nice, I look forward to approving :)

File: test/test_external_libs.py (hunk 3 above)
Comment (emlun) [reply test/test_external_libs.py]: Oooh, now I see why this was there. Good catch, thanks! This should just be updated to use the new environment variable, then.

Comment (emlun) [reply test/test_external_libs.py]: Case in point: all CI build failed... 😁

**Pull request merged**